docs/references.rst

```diff
@@ -0,0 +1,42 @@
+.. SPDX-License-Identifier: CC-BY-4.0
+
+References & Terminology
+========================
+
+Abbreviation
+------------
+
+For the purposes of this document, the following abbreviations apply:
+
+   ================= ===========================================
+   Abbreviation      Description
+   ================= ===========================================
+   FIT               Flat Image Tree
+   FDT               Flattened Device Tree
+   UPL               Universal Payload
+   ================= ===========================================
+
+
+Terminology
+-----------
+
+   ============= =========================================== ============================
+   Term          Definition                                  Examples
+   ============= =========================================== ============================
+   Platform Init Set up the hardware and memory, so that the coreboot, EDK2, OpenSBI,
+                 Payload can interact with that hardware to  Slim Bootloader, U-Boot SPL
+                 prepare for an OS boot. For some projects
+                 this is referred to as bootloader. 
+   Payload       Selects and boots the OS, if required. For  EDK2, Grub, LILO, SeaBIOS,
+                 some projects this is referred as OS Loader U-Boot
```

Comment:
Actually coreboot also treats Grub and LILO as one of payloads, coreboot kinda treat it similar to Linuxboot can directly boot to them:
https://www.coreboot.org/Payloads
Per the definition "As part of system firmware, it mainly initializes boot media and boots the OS", Grub could also technically considered as Payload?

Reply:
From the given link, it looks coreboot supports grub 2 as payload. Since Grub 2 could sit in flash as part of system firmware, or in other boot media to run after other payload (e.g. SeaBIOS). It is good to mention payload is part of system firmware in its definition.

Reply:
Got it, i reworded, could you check if its ok?

Reply:
New change looks good to me.